autoware_map_msgs/msg/PCDMapWithID.msg

```diff
@@ -0,0 +1,3 @@
+# Pointcloud data with ID
+string id
+sensor_msgs/PointCloud2 pcd
```

Comment:
Maybe pointcloud would be better since I think pcd is not common word.
```suggestion
sensor_msgs/PointCloud2 pointcloud
```

Also the file name maybe better 
- PointCloudMapWithID.msg
- IdentifiedPointCloudMap.msg

Reply:
Thanks! I will take `PointCLoudMapWithID.msg`.
Resolved: https://github.com/autowarefoundation/autoware_msgs/pull/39/commits/05b9d7ffc69e5586a1ad3656045f7179c0051a76

Reply:
Just FYI, other message definitions that have `sensor_msgs/PointCloud2` as a field.

```console
❯ rg ^sensor_msgs/PointCloud2 (fd .msg)
./share/map_msgs/msg/PointCloud2Update.msg
5:sensor_msgs/PointCloud2 points

./share/map_msgs/srv/GetPointMapROI_Response.msg
2:sensor_msgs/PointCloud2 sub_map

./share/map_msgs/srv/GetPointMap_Response.msg
2:sensor_msgs/PointCloud2 map

./share/pcl_msgs/msg/PolygonMesh.msg
4:sensor_msgs/PointCloud2 cloud

./share/pcl_msgs/msg/PolygonMesh.msg
4:sensor_msgs/PointCloud2 cloud

./share/map_msgs/msg/PointCloud2Update.msg
5:sensor_msgs/PointCloud2 points

./share/map_msgs/srv/GetPointMapROI.srv
9:sensor_msgs/PointCloud2 sub_map

./share/map_msgs/srv/GetPointMapROI_Response.msg
2:sensor_msgs/PointCloud2 sub_map

./share/map_msgs/srv/GetPointMap_Response.msg
2:sensor_msgs/PointCloud2 map

./share/map_msgs/srv/GetPointMap.srv
3:sensor_msgs/PointCloud2 map
```